print_env/loader.py

```diff
@@ -101,15 +101,23 @@ def load_system(verbose=False):
 
 def load_api(api, token, verbose=False):
     env_vars = {}
+    team = False
```

Comment:
logically this should be fine but in the context of its anticipated data type, I think we should init it as:

```python
team = None
# or
team = ''
```